db/init_db.js

```diff
@@ -9,8 +10,51 @@ async function buildTables() {
     client.connect();
 
     // drop tables in correct order
+    console.log("Dropping tables...")
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS products;
+      DROP TABLE IF EXISTS users;
+    `)
+
 
     // build tables in correct order
+    console.log("Creating tables...")
+    await client.query(`
+      CREATE TABLE products(
+        id SERIAL PRIMARY KEY,
+        name VARCHAR(255) NOT NULL,
+        description TEXT NOT NULL,
+        price INTEGER NOT NULL,
```

Comment:
Maybe change to float, or keep everything as pennies?

Reply:
Made Float.